app/Console/Kernel.php

```diff
@@ -24,7 +24,10 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        $schedule->command('db:refresh') // use custom command to make sure that the commands are chained
+            ->daily() // Run daily at midnight
+            ->environments(['staging', 'local']) // only run for APP_ENV tagged staging or local
```

Comment:
The current dev deploy has `dev` as its environment.

```suggestion
            ->environments(['staging', 'dev', 'local']) // only run for APP_ENV tagged staging, dev, or local
```